### ISSUE TITLE
Failed to run npm run dev

### ISSUE THREAD

**Issue by brendengourley** [michalzaq12/electron-nuxt#14]
**I created a project using the following settings:**
```bash
? Select which css framework install none
? Select which css pre-processor install Sass
? Select with icon set install Font Awesome 5
? Use linting with ESLint? Yes
? Which ESLint config would you like to use? Standard
? Set up unit testing with vue-test-utils + AVA? Yes
? Set up end-to-end testing with Spectron + AVA? Yes
```

**The install worked perfectly, however when running**
```bash
npm run dev
```

**The script failed with the following output:**

```bash
> electron-nuxt@1.0.0 dev /home/brenden/Documents/projects/snapvolt
> cross-env-shell NODE_ENV=development node .electron-nuxt/index.js


  Main  
│  Hash: a54900bab19212206924
│  Version: webpack 4.35.0
│  Time: 807ms
│  Built at: 06/25/2019 9:10:57 AM
│     Asset      Size  Chunks             Chunk Names
│  index.js  63.4 KiB    main  [emitted]  main
│  Entrypoint main = index.js
│  [./.electron-nuxt/config.js] 597 bytes {main} [built]
│  [./node_modules/electron-debug sync recursive] ./node_modules/electron-debug sync 160 bytes {main} [built]
│  [./node_modules/electron-debug/index.js] 2.92 KiB {main} [built]
│  [./node_modules/electron-is-accelerator/index.js] 849 bytes {main} [built]
│  [./node_modules/electron-is-dev/index.js] 256 bytes {main} [built]
│  [./node_modules/electron-localshortcut/index.js] 8.16 KiB {main} [built]
│  [./node_modules/electron-localshortcut/node_modules/debug/src/index.js] 263 bytes {main} [built]
│  [./node_modules/keyboardevent-from-electron-accelerator/index.js] 6.04 KiB {main} [built]
│  [./node_modules/vue-devtools/lib/index.js] 1.35 KiB {main} [built]
│  [./src/main/BrowserWinHandler.js] 1.98 KiB {main} [built]
│  [./src/main/index.dev.js] 969 bytes {main} [built]
│  [./src/main/index.js] 638 bytes {main} [built]
│  [./src/main/mainWindow.js] 498 bytes {main} [built]
│  [electron] external "electron" 42 bytes {main} [built]
│  [path] external "path" 42 bytes {main} [built]
│      + 10 hidden modules
│  
│  WARNING in ./node_modules/electron-debug/index.js 101:45-58
│  Critical dependency: the request of a dependency is an expression
│   @ ./src/main/index.dev.js
│  
│  WARNING in ./node_modules/electron-debug/index.js 102:61-74
│  Critical dependency: the request of a dependency is an expression
│   @ ./src/main/index.dev.js

  Nuxt  
│  ℹ Preparing project for development
│  
│  ℹ Initial build may take a while
│  
│  ✔ Builder initialized
│  
│  ✔ Nuxt files generated
│  
         ✖ Electron-nuxt: [object Object]
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! electron-nuxt@1.0.0 dev: `cross-env-shell NODE_ENV=development node .electron-nuxt/index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the electron-nuxt@1.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/brenden/.npm/_logs/2019-06-25T15_10_59_094Z-debug.log
```

**The content of the debug log:**

```bash
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/local/bin/npm', 'run', 'dev' ]
2 info using npm@6.9.0
3 info using node@v8.11.4
4 verbose run-script [ 'predev', 'dev', 'postdev' ]
5 info lifecycle electron-nuxt@1.0.0~predev: electron-nuxt@1.0.0
6 info lifecycle electron-nuxt@1.0.0~dev: electron-nuxt@1.0.0
7 verbose lifecycle electron-nuxt@1.0.0~dev: unsafe-perm in lifecycle true
8 verbose lifecycle electron-nuxt@1.0.0~dev: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/brenden/Documents/projects/snapvolt/node_modules/.bin:/home/brenden/.sdkman/candidates/kotlin/current/bin:/home/brenden/.sdkman/candidates/gradle/current/bin:/usr/local/pgsql/bin:/home/brenden/.local/bin:/home/brenden/bin:/usr/local/bin:/home/brenden/.local/kitty.app/bin:/home/brenden/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/brenden/Programs/java/jre1.8.0_211/bin:/home/brenden/Programs/jdk1.8.0_211
9 verbose lifecycle electron-nuxt@1.0.0~dev: CWD: /home/brenden/Documents/projects/snapvolt
10 silly lifecycle electron-nuxt@1.0.0~dev: Args: [ '-c',
10 silly lifecycle   'cross-env-shell NODE_ENV=development node .electron-nuxt/index.js' ]
11 silly lifecycle electron-nuxt@1.0.0~dev: Returned: code: 1  signal: null
12 info lifecycle electron-nuxt@1.0.0~dev: Failed to exec dev script
13 verbose stack Error: electron-nuxt@1.0.0 dev: `cross-env-shell NODE_ENV=development node .electron-nuxt/index.js`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at EventEmitter.emit (events.js:214:7)
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at ChildProcess.emit (events.js:214:7)
13 verbose stack     at maybeClose (internal/child_process.js:925:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
14 verbose pkgid electron-nuxt@1.0.0
15 verbose cwd /home/brenden/Documents/projects/snapvolt
16 verbose Linux 4.18.0-22-generic
17 verbose argv "/usr/bin/node" "/usr/local/bin/npm" "run" "dev"
18 verbose node v8.11.4
19 verbose npm  v6.9.0
20 error code ELIFECYCLE
21 error errno 1
22 error electron-nuxt@1.0.0 dev: `cross-env-shell NODE_ENV=development node .electron-nuxt/index.js`
22 error Exit status 1
23 error Failed at the electron-nuxt@1.0.0 dev script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by montoulieu** [comment]
Getting the same issue after npm install and run dev

**Comment by MigFerreira** [comment]
+1
Same here.

**Comment by michalzaq12** [comment]
Fixed in #19

**Comment by MigFerreira** [comment]
Iconset is being incorrectly setup.
when picking fontawesome icons.js is being setup for material icons, and when picking material icons its being setup for fontawesome.
